spec.emu

```diff
@@ -1119,6 +1119,8 @@ contributors: Ron Buckton, Ecma International
               1. Set _completion_ to ThrowCompletion(_error_).
             1. Else,
               1. Set _completion_ to _result_.
+        1. NOTE: After _disposeCapability_ has been disposed, it will never be used again. The contents of _disposeCapability_.[[DisposableResourceStack]] can be discarded at this point.
```

Comment:
```suggestion
        1. NOTE: After _disposeCapability_ has been disposed, it will never be used again. The contents of _disposeCapability_.[[DisposableResourceStack]] can be discarded in implementations, such as by garbage collection, at this point.
```